core/package.json

```diff
@@ -13,7 +13,8 @@
     "./helpers": "./dist/helpers/index.js"
   },
   "scripts": {
-    "build": "tsc --build"
+    "build": "tsc --build",
+    "test": "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/*.spec.ts"
```

Comment:
ts-mocha is suffient, try not to use global executable

Reply:
Resolve in https://github.com/coin98/solana-support-library/pull/1/commits/62a7d352f1fe0fab99693e956bb10cf21e8fe843